### DISCUSSION TITLE
Extend into Ground Distance

### DISCUSSION THREAD

**Discussion by DeadCandy** [GriefPrevention/GriefPrevention#2110] [Q&A]
Hello, I have a question about the config. I see there is an area "Extend in ground distance" and I assume that means the Claim will only go that far? 

I have mine set to 25 however the claims go down to bedrock. I do not want this because when people are mining they wont be able to mine anything down there if they come across some ones claim.

**Comment by Jikoo** [comment]
That's the distance the claim will extend below its creation point by default. You're after the MinimumDepth node.
